lib/train/transports/winrm.rb

```diff
@@ -45,6 +45,9 @@ class WinRM < Train.plugin(1)
     option :port
     option :user, default: 'administrator', required: true
     option :password, nil
+    option :transport, default: :negotiate
```

Comment:
It would be great if you could add a test file in `test/windows/` to exercise the new options (those tests are run by `rake test:windows`, see Rakefile for details).  If that would be overly difficult to do, I'm OK with omitting that for now - but please open a new issue on Train mentioning the need to add testing for the new winrm options, so it doesn't get lost.

Reply:
resolved.